kythe/java/com/google/devtools/kythe/analyzers/java/JavaIndexerConfig.java

```diff
@@ -118,6 +123,13 @@ public boolean getEmitRefCallOverIdentifier() {
     return emitRefCallOverIdentifier;
   }
 
+  public List<String> getAdditionalJavacFlags() {
+    if (additionalJavacFlags == null) {
+      return new ArrayList<String>();
```

Comment:
Is it possible to set `additionalJavacFlags = new ArrayList<String>()` at the declaration site and avoid the conditional here?

Reply:
done